proxy/src/main/java/com/wavefront/agent/handlers/SpanHandlerImpl.java

```diff
@@ -52,6 +56,8 @@
     this.validationConfig = validationConfig;
     this.validItemsLogger = validItemsLogger;
     this.dropSpansDelayedMinutes = dropSpansDelayedMinutes;
+    this.spanTagCount = Metrics.defaultRegistry().newHistogram(new MetricName(handlerKey.toString(), "",
```

Comment:
use the same pattern as `ReportPointHandlerImpl` to determine the registry to use.